File: PHP Excel Examples/GeneralApiDemo/convertInRequest.php
<?php

use Spire\Cloud\Excel\Sdk\Api\GeneralApi;
use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\ExportFormat;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new GeneralApi($configuration);

$file = "D:/input/charts.xlsx";
$password = null;
$format = ExportFormat::XLSX; //Supported formats : Xlsx, Xlsb, Xls, Ods, Pdf, Xps, Ps, Pcl
$result = $apiInstance->convertInRequest($format, $file, $password);

File: Python Excel Examples/GeneralApiDemo/convertInRequest.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.general_api import GeneralApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.general_api.GeneralApi(configuration)
format = "Pdf"   #Supported formats: Xlsx/Xls/Xlsb/Ods/Pdf/Xps/Ps/Pcl
file = "D:/inputFile/charts.xlsx"
password = ""
result = api.convert_in_request(format,file=file, password=password)

